pages/learn/manage/billing.md

```diff
@@ -4,7 +4,29 @@ title: Billing
 
 # Billing
 
-The billing page can be found in the dropdown at the top-right of the dashboard, directly beneath the account preferences. From this dashboard, you can sign up for a paid plan, update your information, change your plan, monitor your usage, and download invoices. If you have any doubts or questions about the billing information shown for your account, please contact [our sales team][sales].
+Billing is managed through [organizations][organizations]. Navigate to the organization for which you'd like to manage billing, and select "billing" from the left hand side menu.
+
+<img width="189" alt="Screen Shot 2021-03-01 at 4 46 36 PM" src="https://user-images.githubusercontent.com/20308284/109579511-e54ea700-7aad-11eb-9cf7-5648f1fdb897.png">
```

Comment:
I see now that this page inherently used the HTML tags way of adding images. Will find them all and replace. A possible reason to use HTML image tags is tinker with the width of a large image so that it fits right.